src/Logary/Configuration_Config.fs

```diff
@@ -104,6 +105,14 @@ let withMetrics ms conf =
   |> Seq.map Metric.validate
   |> Seq.fold (fun s t -> s |> withMetric t) conf
 
+[<CompiledName "WithMiddleware">]
+let withMiddleware middleware conf =
+  { conf with LogaryConf.middleware = middleware :: conf.middleware }
```

Comment:
Does/will ordering of middlewares matter? This looks to be consistent with your concatenation of multiple middlewares below, but we will want to be explicit about how the ordering affects the application.